zetaclient/chains/bitcoin/observer/observer.go

```diff
@@ -52,7 +53,7 @@ const (
 var _ interfaces.ChainObserver = &Observer{}
 
 // Logger contains list of loggers used by Bitcoin chain observer
-// TODO: Merge this logger with the one in evm
+// TODO(revamp): Merge this logger with the one in evm
```

Comment:
This is being refactored by creating base observer/signer/logger structs.

Reply:
Sounds good, we will remove the comment then